### DISCUSSION TITLE
Getting and Setting Album Art (Image) CoverType

### DISCUSSION THREAD

**Discussion by rosshstarr** [Zeugma440/atldotnet#244] [Q&A] [answered]
Hi.  I would love to see an example showing how to Read the Album Art CoverType (eg: Front Cover, Back Cover, Leaflet Page etc.)
I'm relatively new to this, and any help would be very appreciated. Cheers.

**Comment by Zeugma440** [comment, marked as answer]
Hi and thanks for your interest in ATL.net!

You'll find a lot of useful bits of code in the [snippets section](https://github.com/Zeugma440/atldotnet/wiki/3.-Usage-_-Code-snippets#embeddedPics)

The information about the picture type is the `PictureInfo.PicType` property.

**Reply by rosshstarr** [reply]
Hey.  Thanks for the prompt reply, it’s appreciated.  I’ll check out the PictureInfo.PicType (obvious when you think about it 🤭). Cheers and thanks for supplying and supporting ATL, it’s great. 👍🏻
